packages/fdb-host/src/index.test.ts

```diff
@@ -20,7 +21,12 @@ function wrapPeer(peer: Peer) {
   const parser = new ParseJSON();
   const stringifier = new StringifyJSON();
   parser.pipe(peer).pipe(stringifier);
-  return duplexify(parser, stringifier);
+  const nodeVer = semver.parse(process.versions.node);
+  if (nodeVer && nodeVer.major < 16) {
+    return duplexify(parser, stringifier);
+  } else {
+    return stream.Duplex.from({ writable: parser, readable: stringifier });
```

Comment:
`duplexify` causes a hang here. I had a quick dig and couldn't figure out why, didn't spend too much time on it since we can remove it when we raise the minimum supported version to Node 16 and can guarantee `stream.Duplex.from` is available.

Reply:
Found that it's Jest's new timers that seem to upset duplexify, re-enabled the legacy timers to avoid this (for now).